src/main/resources/explanations/challenge22_hint.adoc

```diff
@@ -0,0 +1,20 @@
+This challenge is specifically looking at a secret in a Rust binary
+
+You can solve this challenge using the following steps:
+
+TODO
```

Comment:
Let’s do this before merging! But before that: let’s first finish the binary stuff